### PR TITLE
Remove XML tagging from com.netscape.certsrv.system

### PR DESCRIPTION
**Pull request by ckelleyRH**

### CODE REVIEW

File: base/common/src/main/java/com/netscape/certsrv/system/DomainInfo.java [hunk 1]
```diff
@@ -40,16 +29,13 @@
...
-@XmlRootElement(name="DomainInfo")
-@XmlAccessorType(XmlAccessType.NONE)
 @JsonInclude(Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown=true)
 public class DomainInfo implements JSONSerializer {
 
     String name;
     Map<String, SecurityDomainSubsystem> subsystems = new LinkedHashMap<>();
 
-    @XmlAttribute(name="id")
```
Comment (edewata) [inline base/common/src/main/java/com/netscape/certsrv/system/DomainInfo.java:52]: I think we need to replace this with `@JsonProperty("id")` above the `name` field declaration.

Comment (ckelleyRH) [reply base/common/src/main/java/com/netscape/certsrv/system/DomainInfo.java:52]: I'll give it a go, thanks!

Comment (ckelleyRH) [reply base/common/src/main/java/com/netscape/certsrv/system/DomainInfo.java:52]: Might also need it on the subsystem array, but first I'll try removing all the property tags to get a feel for what is happening.

Comment (ckelleyRH) [reply base/common/src/main/java/com/netscape/certsrv/system/DomainInfo.java:52]: I added in `@JsonProperty("id")` and the key ended up being `@id`, which leaves me suspicious that this annotations might be causing more harm than good, but we will see when the CI completes.

Comment (ckelleyRH) [reply base/common/src/main/java/com/netscape/certsrv/system/DomainInfo.java:52]: OK, I think I see what is happening now... putting the annotations back in.

**Comment by ckelleyRH** [comment]
@edewata I think I have found all the fields that need annotating now :+1:

**Review by edewata — APPROVED** [review]
Noice!

**Pull request merged**